pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -248,7 +254,7 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 							{
 								Name:            Druid,
 								Image:           b.image,
-								ImagePullPolicy: corev1.PullIfNotPresent,
+								ImagePullPolicy: corev1.PullAlways,
```

Comment:
I assume this is also a leftover to quickly change etcd versions during testing?